pkg/secret/replicated.go

```diff
@@ -201,13 +194,14 @@ func requiresUpdate(current, desired *replicationapi.ReplicatedSecret) bool {
 	return false
 }
 
-func getManagedObjectMeta(name, namespace, clusterName string) metav1.ObjectMeta {
+func getManagedObjectMeta(name string, kcKey client.ObjectKey) metav1.ObjectMeta {
```

Comment:
This will hopefully remove the risk of confusing `kc.Name` with `kc.Spec.Cassandra.Cluster` 👍